server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -323,11 +324,17 @@ protected Modal buildConfirmCommonIntakeChangeModal(String existingCommonIntakeF
         .build();
   }
 
-  private ButtonTag createSubmitButton(Request request) {
+  private ButtonTag createSubmitButton(Request request, ProgramEditStatus programEditStatus) {
     String saveProgramDetailsText;
-    if (settingsManifest.getProgramCardImages(request)) {
+    if (settingsManifest.getProgramCardImages(request)
+        && (programEditStatus == ProgramEditStatus.CREATION
+            || programEditStatus == ProgramEditStatus.CREATION_EDIT)) {
```

Comment:
suggestion (non-blocking): replace the || with `!ProgramEditStatus.EDIT`

Reply:
I liked this way a little better because I think it's easier to understand "I want the creation statuses" instead of "I don't want the edit status" when reading the code. Using `!ProgramEditStatus.EDIT` means I'll probably need to read the enum to see what the other values there are, which is just sliiightly more work.

Reply:
ahhh gotcha. that makes sense to me!